components/Cta.vue

```diff
@@ -33,13 +34,13 @@ export default {
     return {
       script: [
         {
-          src: "https://app.formester.com/widget/popup.js",
+          src: "https://bevdkbdf.formester.com/widget/popup.js",
```

Comment:
Will this work as this is not survey page?

Reply:
Yes, this is what we currently have in production @anirudh-tantry. Also, you can check the deploy preview [here](https://deploy-preview-40--acorn-globus.netlify.app/) (It seems to be working)

![image](https://github.com/acorn-globus/acorn-globus-site/assets/69912046/837b3330-e2a9-49d4-b50c-48b85202b65c)

![image](https://github.com/acorn-globus/acorn-globus-site/assets/69912046/13e4e7d7-b8f1-45ea-8f5d-fb590699c434)